### PR TITLE
Fix flag icons in phone number input only showing a "corner of a flag"  on IE

### PR DESCRIPTION
**Pull request by elhesuu**
# Problem
Flag icons in phone number input only show a "corner of a flag"  on IE.

IE doesn't scale SVG images without `viewBox` attribute. 

# Solution
Since icons are loaded from another project and we can't add an attribute, the chosen approach is to use a custom `flagComponent` component, rendered as a `span` element with a background image instead of an `img` tag.

## Checklist
- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a ] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-342-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-342-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://342-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/PhoneNumberInput/index.js [hunk 1]
```diff
@@ -36,6 +46,7 @@ class PhoneNumberInput extends Component {
         country={this.state.country}
         inputClassName={`${style.mobileInput}`}
         className={`${style.phoneNumberContainer}`}
+        flagComponent={ FlagComponent }
```
Comment (stefaniacardenas) [inline src/components/PhoneNumberInput/index.js]: Hi @elhesuu, nice to meet you.
Looks good. I didn't know the library had this prop. 👍

Comment (elhesuu) [reply src/components/PhoneNumberInput/index.js]: Nice to meet you too @stefaniacardenas !

**Pull request merged**